lib/db/mem.js

```diff
@@ -104,6 +112,11 @@ module.exports = function (log, error) {
       lastAccessTime: sessionToken.createdAt
     }
 
+    unverifiedTokens[tokenId] = {
+      tokenVerificationId: sessionToken.tokenVerificationId,
+      uid: sessionToken.uid.toString('hex')
```

Comment:
I don't think it matters really, but I'm happy to change if you feel strongly about it.

I decided to treat them differently because `unverifiedTokens.uid` is never returned but `unverified.tokenVerificationId` is. It makes sense to store the string for the former because it is only used in string comparisons. But for `tokenVerificationId` a choice must be made between storing it as a buffer and creating comparison strings on the fly, or storing it as a string and creating the return buffer on the fly.

The convention for other fields seemed to be to store them as buffers, so I chose that option. But it's just as easy to flip it the other way.